src/rust/engine/process_execution/src/lib.rs

```diff
@@ -130,6 +130,11 @@ pub struct ExecuteProcessRequest {
   #[derivative(PartialEq = "ignore", Hash = "ignore")]
   pub description: String,
 
+  // This will be materialized for local ExecuteProcessRequest only.
+  // Eventually we want to remove this.
+  // Context: https://github.com/pantsbuild/pants/pull/8282
+  // Think twice before using it.
+  pub local_only_scratch_files: hashing::Digest,
```

Comment:
This field should be ignored for the purposes of node equality - please modify the existing `execute_process_request_equality` test to show that it is ignored.

Reply:
Test added